config/redirects

```diff
@@ -226,3 +226,9 @@ raw: docs/drivers/drivers/driver-compatibility-reference -> ${base}/
 # Redirect for outdated Community Drivers page
 
 raw: docs/drivers/community-supported-drivers/ -> ${base}/
+
+# Redirects for consistent language-spcific driver landing pages
+
+raw  docs/drivers/php/ -> ${base}/php-drivers/
+raw  docs/drivers/python/ -> ${base}/python-drivers/
+raw  docs/drivers/ruby/ -> ${base}/ruby-drivers/
```

Comment:
Issue: 
I could be wrong, but I think these need a colon after "raw". These redirects also didn't surface in the mut-redirects output diff, so I am guessing they were ignored.

Suggestion:

```suggestion
raw: docs/drivers/php/ -> ${base}/php-drivers/
raw: docs/drivers/python/ -> ${base}/python-drivers/
raw: docs/drivers/ruby/ -> ${base}/ruby-drivers/
```